modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/resource/v1_0/NodeMetricResourceImpl.java

```diff
@@ -220,18 +220,14 @@ private BooleanQuery _createNodesBooleanQuery(
 				_queries.wildcard("name", "*" + key + "*"));
 		}
 
-		TermsQuery termsQuery = null;
+		TermsQuery termsQuery = _queries.terms("name");
 
 		if (!taskNames.isEmpty()) {
-			termsQuery = _queries.terms("name");
```

Comment:
Hey @MarinhoFeliphe, I think we can remove this if. What do you think? Thx

Reply:
Hi @rafaprax, maybe it will can to cause a NullPointerException, I simplified to other way. Whato do you think?